api/internal/handler/ssl/ssl.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/shiningrush/droplet/wrapper"
 	wgin "github.com/shiningrush/droplet/wrapper/gin"
 
+	"github.com/apisix/manager-api/conf"
```

Comment:
we'll move this module to another place, is it right?

Reply:
yes. when we adjust the code dir.